lib/core/src/lib/datatable/components/datatable/datatable.component.scss

```diff
@@ -229,6 +229,8 @@ $data-table-cell-min-width-file-size: $data-table-cell-min-width !default;
         box-sizing: border-box;
 
         .adf-datatable-row {
+            padding-right: 15px;
```

Comment:
<img width="1718" alt="Zrzut ekranu 2023-07-11 o 09 12 21" src="https://github.com/Alfresco/alfresco-ng2-components/assets/113341662/2b997cc4-c0f1-4d1d-a74d-718dacdcd900">
<img width="1728" alt="Zrzut ekranu 2023-07-11 o 09 16 08" src="https://github.com/Alfresco/alfresco-ng2-components/assets/113341662/4058f5d2-aebc-4a84-b0f0-64b959eb2b76">
I will say it one more time, please do test your changes before you commit them! Above there are 2 screenshots from control-center and ACA with broken column layout caused by your changes... Apparently you've just commited those without even checking if it works..

Reply:
@arohilaGL spend time to test this and please ask for review after you do this. @rbahirsheth please provide support here.

Reply:
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/4d878947-aeb9-446b-8de7-3d6a78598e4c)
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/96c4d4bd-4235-46f1-8943-d968409b21e7)
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/53c0d783-0462-4eeb-88fd-86eeb3180e4f)
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/f7527bef-a4a4-4792-8b91-01331749278d)
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/edd4660a-4290-4324-8ad0-faa619a8df1d)

Reply:
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/c423f7ac-bc1a-4474-956d-882e470a4376)
ACA>Personal Files

Reply:
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/c6889fdf-8f7b-47c4-8e1e-7b7bb9c70d63)
ACA > Personal File

Reply:
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/4946705b-0a31-4ae1-8b01-73625740a9e4)
ACA > Recent Files

Reply:
That means that solution you provided doesn't work for different OS or browser, we can't accept solutions that breaks layout somewhere

Reply:
@MichalKinas , can you please share browser version and OS, because we have validated our changes in MAC OS (chrome, safari and firefox), windows(chrome and firefox), linux( chrome and firefox). I am attaching a few screenshots. And the changes are working fine.

safari MacOS
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/0b261696-2f86-4f70-a85b-a12b20d7eaa8)
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/46242145-33a8-48fc-8adc-48d887d6592f)

Firefox MacOS
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/ffcbbffe-b550-4d8c-9c94-1b9ea926041b)
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/c67dcd37-d720-4b16-bb11-6272bd4cd1ea)


Chrome MacOS

![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/e871526e-1a0d-455c-aafd-e14598712b53)
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/31a2b1eb-a190-44b2-a54a-832fd16a9975)




Linux Firefox Version 115.0

![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/b6de52ef-21c8-45aa-9c74-92d78b0636df)
 Linux Chrome Version 114.0.5735.198

![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/14210503-074e-4e20-aa91-ef33623167fb)

If needed, I can provide you rest of the browser versions as well.
A few more screenshots have been attached above.

Reply:
So my MacOS is `13.4.1 (22F82)` and Chrome `Version 114.0.5735.198 (Official Build) (arm64)`

Reply:
@AleksanderSklorz could you please test it on your browser?

Reply:
@MichalKinas , we tested it on MacOS and Chrome Version 114.0.5735.198 (Official Build), I have attached the screenshots above for the same. Maybe someone else can test it on their browser. @AleksanderSklorz , that would be a lot of help, thanks.

Reply:
Hi @MichalKinas @MateuszHY @arohilaGL I checked it quickly and I noticed that it looks a bit different depending on zoom/resolution: 

100% zoom: 
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/f404bb36-5659-4fb6-8837-78d7dbdbe844)

50% zoom: 
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/5d06a01c-9f35-44c0-bb37-74d038ff3185)


You can see that for example values for "Modified by" are a bit shifted

Reply:
I think what actually happend on my mac is described here https://www.lifewire.com/change-how-macs-scroll-bars-work-2260834#:~:text=Go%20to%20the%20Apple%20menu,%2C%20When%20Scrolling%2C%20or%20Automatically.

Reply:
Hi @AleksanderSklorz 
I checked this issue & this is because of overflow-y style is applied & due to this is shifted a bit to the left. Please check the screenshot below.
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/111955585/7df9ba65-b462-405c-bdc6-06e64c02d3eb)
I have created a new ticket. https://alfresco.atlassian.net/browse/ADF-5555
We will fix this issue later.
Can we do this if you agree? What do you think?

Reply:
@rbahirsheth please also see @MichalKinas comment above. There is similar problem I guess which can be fixed together with that. @MichalKinas do you agree?

Reply:
btw @rbahirsheth please assign that jira to your team

Reply:
Sure, Aleksander. I will assign it, but need to discuss it with Dona.

Reply:
Hi, @MichalKinas @AleksanderSklorz Please approve this PR if you don't find any issue in the PR. I will discuss with Dona the newly created ticket.
Thanks.